packages/frontend/src/components/AudioRecorder/AudioRecorder.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useRef, useState } from 'react'
+import MicRecorder from './MicRecorder'
+import styles from './styles.module.scss'
+
+type MicRecorderType = {
+  start: () => Promise<MediaStream>
+  stop: () => MicRecorderType
+  getMp3: () => Promise<[buffer: any, blob: any]>
+}
+
+export const AudioRecorder = ({
+  recording,
+  setRecording,
+  saveVoiceAsDraft,
+}: {
+  recording: boolean
+  setRecording: (f: boolean) => void
+  saveVoiceAsDraft: (blob: Blob) => void
+}) => {
+  const recorder = useRef<MicRecorderType>(
+    new MicRecorder({
+      bitRate: 128,
+    })
+  )
```

Comment:
Again, this will re-instantiate the `MicRecorder` on every re-render. I don't know how expensive it is.